GitCommands/Settings/AppSettings.cs

```diff
@@ -369,6 +369,12 @@ public static bool ShowGitStatusInBrowseToolbar
             set => SetBool("showgitstatusinbrowsetoolbar", value);
         }
 
+        public static bool ShowGitStatusForArtificialCommits
+        {
+            get => GetBool("showgitstatusForArtificialCommits", true);
+            set => SetBool("showgitstatusForArtificialCommits", value);
```

Comment:
If this is a new setting name, please make it either all in lower-case or in PascalCase. Otherwise it looks odd